content/source/docs/enterprise/run/cli.html.md

```diff
@@ -126,7 +126,7 @@ Remote applies use the configuration code from the local working directory, but
 $ terraform apply
 
 Running apply in the remote backend. Output will stream here. Pressing Ctrl-C
-will cancel the remote apply if its still pending. If the apply started it
+will cancel the remote apply if it's still pending. If the apply started it
```

Comment:
```suggestion
will cancel the remote apply if it's still pending. If the apply has started, it
```

Reply:
i'm going to _sadly_ reject the change because the TF PR was merged without this very good and correct suggestion.  BUT i encourage someone else (or me) to make this change soon!